client/src/components/ProTx/components/util.js

```diff
@@ -1,7 +1,40 @@
 import { getColor } from './maps/intervalColorScale';
-import { THEME_CB12_MAIN } from './colors';
-import { PHR_MSA_COUNTIES } from './PHR_MSA_County_masterlist';
-import { OBSERVED_FEATURES, MALTREATMENT } from './meta';
+import { THEME_CB12_MAIN, THEME_CB12_ALT0 } from './colors';
+import PHR_MSA_COUNTIES from './PHR_MSA_County_Data';
+import { OBSERVED_FEATURES, MALTREATMENT, CATEGORY_CODES } from './meta';
+
+/**
+ * Assign an imported color theme for use in plot generation.
+ */
+export const plotColors = THEME_CB12_MAIN;
+export const histColors = THEME_CB12_ALT0;
```

Comment:
maybe those don't need to be exported?  like in getTraceFillColor, just `THEME_CB12_ALT0` could also be directly used.

Reply:
We could certainly drop the exports. There are a number of methods that could use the same cleanup (never used outside utils.js). Lets grab those in a cleanup ticket.

I want to keep the color array variable assignments so I can swap out the plot "themes" in the trace fill method without having to touch the internal plumbing.